servicemap/rest_client.py

```diff
@@ -8,12 +8,23 @@ class ServicemapApiClient:
     def __init__(self, config) -> None:
         self.root = config["ROOT"]
 
-    def list_helsinki_schools_and_kindergartens(self, filter_list=None):
+    def list_helsinki_schools_and_kindergartens(self, filters=None):
```

Comment:
How about adding type hint to filters?
```suggestion
    def list_helsinki_schools_and_kindergartens(self, filters: Optional[dict] = None):
```